safeweb/http_test.go

```diff
@@ -561,3 +562,50 @@ func TestGetMoreSpecificPattern(t *testing.T) {
 		})
 	}
 }
+
+func TestStrictTransportSecurityOptions(t *testing.T) {
+	tests := []struct {
+		name          string
+		options       string
+		secureContext bool
+		expect        string
+	}{
+		{
+			name: "off by default",
+		},
+		{
+			name:          "default HSTS options in the secure context",
+			secureContext: true,
+			expect:        defaultStrictTransportSecurityOptions,
+		},
+		{
+			name:          "custom options sent in the secure context",
+			options:       "max-age=31536000; includeSubDomains",
```

Comment:
I wonder if we ought not export the default so that someone can more easily include it with other options, since it seems like you'd usually want the max-age regardless?